x/halo/keeper/msg_server_aggregator.go

```diff
@@ -48,7 +48,7 @@ func (k aggregatorMsgServer) ReportBalance(goCtx context.Context, msg *aggregato
 	}
 	k.SetRound(ctx, id, round)
 
-	if msg.NextPrice.IsZero() || msg.NextPrice.LTE(answer) {
+	if msg.NextPrice.IsNegative() || msg.NextPrice.IsZero() || msg.NextPrice.LTE(answer) {
```

Comment:
In theory could we simply this to `!msg.NextPrice.IsPositive()`?

That would ensure that it's both not negative and not zero 😄